docs/cheat-sheet.md

```diff
@@ -271,6 +271,20 @@ fn.spec.min_replicas = 1
 fn.spec.max_replicas = 4
 ```
 
+#### Scale to zero
+
+```python
+# Nuclio/serving scaling
+fn.spec.min_replicas = 0    # zero value is mandatory for scale to zero
+fn.spec.max_replicas = 2
+
+# Scaling to zero in case of 20 minutes (idle-time duration)
+fn.set_config(key="spec.scaleToZero.scaleResources",
+              value=[{"metricName":"nuclio_processor_handled_events_total",
+                      "windowSize" : "20m",
+                      "threshold" : 0}])
```

Comment:
these are true for mlrun deployed with iguazio of course.
for open source, it is up to the user to set up prometheus stack, scrape and publish to kube metrics server the metric name and window sizes.

for the record, on iguazio deployments, by default, the window sizes available are - 1m, 2m, 5m, 10m, 30m